pkg/handler/topology.go

```diff
@@ -234,7 +234,7 @@ func expandQueries(queries filters.MultiQueries, namespace string, isForProm fun
 	}
 	q2 := filters.SingleQuery{
 		filters.NewMatch(fields.FlowDirection, `"`+string(constants.Egress)+`"`),
-		filters.NewMatch(fields.DstOwnerName, `""`),
+		filters.NewMatch(fields.DstOwnerType, `"","Service"`),
```

Comment:
in fact we could use:

```suggestion
		filters.NewMatch(fields.DstType, `"","Service"`),
```

.. it works the same, but with the advantage to not have to add another index (it's already indexed)

Reply:
Here we go
https://github.com/netobserv/network-observability-console-plugin/pull/580/commits/609a6068e9e683e0d09bda295669976661034d4c